### DISCUSSION TITLE
Error translating key "ui.common.week" in language "pl": TypeError: Cannot read properties of undefined (reading 'ui')

### DISCUSSION THREAD

**Discussion by modo1302** [totaldebug/atomic-calendar-revive#1421] [General]
Hey, while in browser's console i get lots and lots of errors. Is this normal?
![image](https://github.com/totaldebug/atomic-calendar-revive/assets/133964285/9ad9cb39-cc30-4b01-a05a-3e858a4f5e6a)

**Comment by es3nric** [comment]
Same here with locale "es"

**Comment by marksie1988** [comment]
This should be fixed in the latest release.
